amp_conf/htdocs/admin/libraries/BMO/Ajax.class.php

```diff
@@ -63,14 +63,15 @@ public function doRequest($module = null, $command = null) {
 			$this->ajaxError(403, 'ajaxRequest declined');
 		}
 
+		$url = parse_url($_SERVER['HTTP_REFERER']);
```

Comment:
REMOTE_ADDR is always an IP address. http://php.net/manual/en/reserved.variables.server.php